### DISCUSSION TITLE
blink> build up n Response(     ErrorResponse {         status_code: 409,         message: "The disk and other machine associated resources are still in use. If a machine recently shut down, please wait.",     }, )

### DISCUSSION THREAD

**Discussion by oliverv** [blinksh/blink#2029] [Support]
### Checklist

- [ ] Using latest version on the [App Store](http://itunes.apple.com/app/id1156707581)
- [ ] Read the [docs](http://blink.sh/docs/)
- [ ] Searched for [existing GitHub issues](https://github.com/blinksh/blink/issues)

**Comment by carloscabanero** [comment]
Hi! This can happen if you start a machine and subsequently try to start it again while the first one is already in the process of booting up. Although usually the whole operation takes just a few seconds, if there are no resources available it may take a bit longer.

If you are still experiencing issues, please open a support request by sending an email to support at blink.build.
